app/src/renderer/components/common/ActionModal.vue

```diff
@@ -126,50 +147,73 @@ export default {
     }
   },
   data: () => ({
+    step: `txDetails`,
     signMethod: null,
     password: null,
-    selectedSignMethod: `local`,
-    signMethods: [
-      {
-        key: `Account Password (Unsafe)`,
-        value: `local`
-      }
-      // {
-      //   key: `Ledger`,
-      //   value: `ledger`
-      // },
-      // {
-      //   key: `Cosmos Signer App`,
-      //   value: `signer-app`
-      // }
-    ],
     sending: false,
     submissionError: null,
     show: false
   }),
   computed: {
-    ...mapGetters([`connected`])
+    ...mapGetters([`connected`, `ledger`]),
+    selectedSignMethod() {
+      if (this.ledger.isConnected) {
```

Comment:
we also could  check for the locally signed in account type

Reply:
ie `if (this.ledger.isConnected && user.signedIn) {` ?

Reply:
but action modals won't be opened if the user hasn't logged in, so 🤷‍♂️

Reply:
i think they should open but display a message that says you need to sign in to use them.

Reply:
> i think they should open but display a message that says you need to sign in to use them.

hmm true, maybe for the next iteration ?